src/panels/config/integrations/integration-panels/dynalite/dynalite-config-panel.ts

```diff
@@ -0,0 +1,289 @@
+                  type="number"
+                  .value=${this._fade}
+                  @value-changed=${this._handleChange}
+                ></paper-input>
+              </ha-settings-row>
+            </div>
+            <div class="card-content">
+              <ha-settings-row .narrow=${this.narrow}>
+                <span slot="heading">${this.localStr("active")}</span>
+                <span slot="description">${this.localStr("active_long")}</span>
+                <ha-paper-dropdown-menu
+                  label=${this.localStr("active")}
+                  dynamic-align
+                >
+                  <paper-listbox
+                    slot="dropdown-content"
+                    .selected=${this._activeIndex}
+                    @iron-select=${this._handleActiveSelection}
+                  >
+                    ${this._activeOptions.map(
+                      (option) =>
+                        html`<paper-item .active_config=${option}
+                          >${this.localStr("active_" + option)}</paper-item
+                        >`
+                    )}
+                  </paper-listbox>
+                </ha-paper-dropdown-menu>
+              </ha-settings-row>
+            </div>
+            <div class="card-content">
+              <ha-settings-row .narrow=${this.narrow}>
+                <span slot="heading">${this.localStr("auto_discover")}</span>
+                <span slot="description"
+                  >${this.localStr("auto_discover_long")}</span
+                >
+                <ha-switch
+                  .checked=${this._auto_discover}
+                  @change=${this._handleAutoDiscoverChange}
+                ></ha-switch>
+              </ha-settings-row>
+            </div>
+            <div class="card-content">
+              <ha-settings-row .narrow=${this.narrow}>
+                <span slot="heading">${this.localStr("poll_timer")}</span>
+                <span slot="description"
+                  >${this.localStr("poll_timer_long")}</span
+                >
+                <paper-input
+                  class="flex"
+                  .label=${this.localStr("poll_timer")}
+                  name="poll_timer"
+                  type="number"
+                  .value=${this._poll_timer}
+                  @value-changed=${this._handleChange}
+                ></paper-input>
+              </ha-settings-row>
+            </div>
+            <div class="card-actions">
+              <mwc-button @click=${this._publish}>
+                ${this.localStr("publish")}
+              </mwc-button>
+            </div>
+          </ha-card>
+        </div>
+      </ha-app-layout>
+    `;
+  }
+
+  protected async firstUpdated() {
+    const configEntryId = this._getConfigEntry();
+    if (!configEntryId) return;
+    const response = await this.hass.callWS({
+      type: "dynalite/get_entry",
+      entry_id: configEntryId,
+    });
+    this._entryData = (response as any).data;
```

Comment:
add typing instead of `any` this

Reply:
what type should i expect from the websocket call? should i define it as a custom type?

Reply:
what you are sending in from the backend, yes, you should define the interface.

Reply:
done